.circleci/config.yml

```diff
@@ -198,25 +198,25 @@ workflows:
       - lint:
           name: Lint
           context: nodejs-install
-          node_version: "16.13.2"
+          node_version: "18.19.1"
       - test-unix:
           name: Unix with node v<< matrix.node_version >>, elixir v<< matrix.elixir_version >>
           context: nodejs-install
           requires:
             - Lint
           matrix:
             parameters:
-              node_version: ["12.22.11", "14.20.0", "16.13.2"]
-              elixir_version: [ "1.14.1", "1.13.1", "1.12.1"]
+              node_version: ["18.19.1"]
+              elixir_version: [ "1.17-erlang-27.2.2", "1.18-erlang-27.2.2"]
```

Comment:
The cimg tags seems to not work for pure elixir versions since this was last done: https://circleci.com/developer/images/image/cimg/elixir#image-tags